cluster-up/check.sh

```diff
@@ -53,8 +53,10 @@ if is_enabled "$KVM_NESTED"; then
 	echo "[ OK ] $KVM_ARCH nested virtualization enabled"
 else
 	echo "[ERR ] $KVM_ARCH nested virtualization not enabled"
+	exit 1
```

Comment:
/hold
i am not sure it is fine to do so, if kubevirtci supports emulation then it shouldnt exit

please verify that deeply first

Reply:
Started a discussion in the mailing list. Thanks :)

Reply:
thanks, imho slack / this PR can fit better for such discussion
also if using mailing list, worth to add the PR link please

Reply:
Closing it based on the discussion in the mailing list.